strings/base_implements.h

```diff
@@ -1218,6 +1218,56 @@ namespace winrt::impl
     };
 #endif
 
+    template<typename T>
+    class has_initialize_instance_member
```

Comment:
Do we really need this level of flexibility? member vs free? Why not keep it as simple as possible?

Reply:
Yes, could remove member (free is required to avoid a coordinate change with Xaml compiler).